helm/prometheus-rules/templates/recording-rules/cortex.rules.yml

```diff
@@ -46,7 +46,7 @@ spec:
       record: aggregation:giantswarm:api_requests
     - expr: count(app_operator_app_info{status="deployed",namespace="giantswarm"}) by (app,name,version,catalog)
       record: aggregation:giantswarm:app_deployed_management_cluster_total
-    - expr: count(app_operator_app_info{status=~"deployed|DEPLOYED",namespace!="giantswarm"}) by (app,name,version,catalog)
+    - expr: count(app_operator_app_info{status=~"deployed|DEPLOYED",namespace!="giantswarm"}) by (app,app_version,name,version,catalog)
```

Comment:
This is to show the upstream chart version in the new Managed Apps Usage dashboard.

See https://github.com/giantswarm/dashboards/pull/63